mesonbuild/interpreter/primitives/string.py

```diff
@@ -108,6 +108,7 @@ def join_method(self, args: T.Tuple[T.List[str]], kwargs: TYPE_kwargs) -> str:
         return self.held_object.join(args[0])
 
     @noKwargs
+    @FeatureNew('str.replace', '0.58.0')
```

Comment:
This actually used to have a check but in commit d93d01b6c5c98a77e057b914e8d66b01d2a10771 while refactoring to create primitives/string.py, it got accidentally dropped. Thanks for the catch!